src/oracle.py

```diff
@@ -0,0 +1,135 @@
+from time import sleep
```

Comment:
I'd like to offer to rename "oracle.py" into "main.py". 
I got it from the Dockerfile. "oracle.py" is a start point of all oracle applications and singular. Let's name it with a strict name correlation.